config/initializers/datadog.rb

```diff
@@ -17,6 +17,9 @@
       }
     end
 
+    # Enable trace-log correlation
+    c.tracing.log_injection = true
```

Comment:
i am unsure if any of the other traces using `Datadog::Tracing` are actually being traced without this enabled 🤔 
happy to take this out if this is not needed, of course!

Reply:
@rmtolmach thanks for the documentation. i can take this line out and see if it is already working!